src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -549,7 +551,9 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
             }
             catch (Exception x) {
                 LOGGER.log(Level.WARNING, "Failed to transform POM; continuing", x);
-                pomData.getWarningMessages().add(Functions.printThrowable(x));
+                StringWriter sw = new StringWriter();
+                x.printStackTrace(new PrintWriter(sw));
+                pomData.getWarningMessages().add(sw.toString());
```

Comment:
Rather than promote Guava to a full-fledged dependency I rewrote this without Guava. I am deleting it in another PR anyway so it doesn't matter.